src/ui/mcp/oauth/security_test.go

```
@@ -0,0 +1,239 @@
+		"code_challenge_method": {"S256"},
+		"resource":              {testResource},
+		"scope":                 {"mcp"},
+		"username":              {"user"},
+		"password":              {"secret"},
+	}
+	resp := postForm(t, app, "/oauth/authorize", form)
+	require.Equal(t, fiber.StatusBadRequest, resp.StatusCode)
+	assertOAuthError(t, resp, "invalid_request")
+
+	var codes int
+	require.NoError(t, srv.store.db.QueryRow(`SELECT COUNT(*) FROM oauth_authorization_codes`).Scan(&codes))
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).QueryRow must not be called. use (*database/sql.DB).QueryRowContext

(noctx)